droidlet/tools/artifact_scripts/upload_artifacts_to_aws.py

```diff
@@ -14,6 +14,82 @@
 print("Rootdir : %r" % ROOTDIR)
 
 
+def tar_and_upload(agent, artifact_name, model_name=None):
```

Comment:
I don't understand - why do we still have another function to do this ?

Reply:
Instead of adding a new function that redundantly does a part of `compute_checksum_tar_and_upload`, can we extract out the shared functionality of the two functions (actually tar'ing and uploading) and create a new method that both these methods call ?

Reply:
It is a good idea. I will refactor ```compute_checksum_tar_and_upload```.

Reply:
@kavyasrinet 
I've refactored the previous method ```compute_checksum_tar_and_upload```, which consists of mainly two parts (1.compute checksum, 2.tar and upload). Since computing checksum has been implemented in ```compute_checksum.py```, I directly import from it. Then I move tar and upload out as an independent method. Lastly, I rename ```compute_checksum_tar_and_upload``` to ```upload_artifacts_to_aws```. Please let me know if it is reasonable.